src/cms/views/media/media_list_view.py

```diff
@@ -1,42 +1,67 @@
+"""
+The module is working as the view component for the dynamic data loading for the media library.
+Therefore, it's managing the region permissions and connects the different data structures.
+Especially, the root file, the use of the file defined in the Document and the different meta data.
+"""
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render
 from django.utils.decorators import method_decorator
 from django.views.generic import TemplateView
 
-from ...decorators import region_permission_required
-from ...models import Document
+from .content_media_mixin import ContentMediaMixin
+
+from ...decorators import region_permission_required, staff_required
 
 
 @method_decorator(login_required, name="dispatch")
 @method_decorator(region_permission_required, name="dispatch")
-class MediaListView(TemplateView):
+class MediaListView(TemplateView, ContentMediaMixin):
     """
-    View for listing media elements
+    Class representing the media management and renders the dynamic data into the HTML template.
     """
 
-    #: The template to render (see :class:`~django.views.generic.base.TemplateResponseMixin`)
     template_name = "media/media_list.html"
     #: The context dict passed to the template (see :class:`~django.views.generic.base.ContextMixin`)
     base_context = {"current_menu_item": "media"}
 
     def get(self, request, *args, **kwargs):
         """
         Render media list
+        """
+        context = self.get_context_data(**kwargs)
 
```

Comment:
```suggestion
    extra_context = {"current_menu_item": "media"}
```
No reason to over-complicate things.